spring-boot-project/spring-boot-dependencies/pom.xml

```diff
@@ -118,7 +118,7 @@
 		<kafka.version>1.1.0</kafka.version>
 		<kotlin.version>1.2.41</kotlin.version>
 		<lettuce.version>5.1.0.M1</lettuce.version>
-		<liquibase.version>3.6.1</liquibase.version>
+		<liquibase.version>3.6.2-SNAPSHOT</liquibase.version>
```

Comment:
Please notice this was used just to test, we must remove the `SNAPSHOT`as soon as the new versions is available